importer/Import/Repodata.hs

```diff
@@ -94,7 +94,11 @@ loadRepoFromURI uri = do
 
  where
     appendOrThrow :: T.Text -> URI
-    appendOrThrow path = appendURI uri (T.unpack path) `throwIfNothing` RepoException
+    appendOrThrow path = let
+          -- If the URI path already ends in a slash, don't add an extra one
+          new_path = if last (uriPath uri) == '/' then T.unpack path else "/" ++ T.unpack path
```

Comment:
Can you post the exact steps leading to this problem?  I haven't seen it, but I haven't tried too hard to reproduce it either.  Thanks.